core/src/main/scala/org/broadinstitute/dsp/workbench/welder/StorageLinksInterp.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsp.workbench.welder
+
+import cats.effect.IO
+
+class StorageLinksInterp(storageLinksCache: StorageLinksCache) extends StorageLinksAlg {
+  def findStorageLink[A](localPath: RelativePath): IO[CommonContext] = for {
+    storageLinks <- storageLinksCache.get
+    baseDirectories = getPosssibleBaseDirectory(localPath.asPath)
+    context = baseDirectories.collectFirst {
+      case x if (storageLinks.get(x).isDefined) =>
```

Comment:
Checking: this map is indexed by both the edit mode base dir and the safe mode base dir?

Reply:
yes...see https://github.com/DataBiosphere/welder/blob/master/server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksService.scala#L39